packages/transport-ble/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@aries-framework/transport-ble",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.1.0",
+  "files": [
+    "build"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "description": "Bluetooth Low Energy transport for Aries Framework JavaScript",
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript-ext/tree/main/packages/transport-ble",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript-ext",
+    "directory": "packages/transport-ble"
+  },
+  "scripts": {
+    "sample": "tsnd --respawn samples/sample.ts",
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@animo-id/react-native-ble-didcomm": "^0.4.1",
+    "@aries-framework/core": "^0.3.2",
+    "react-native": "^0.70.6"
+  },
+  "devDependencies": {
+    "@types/react-native": "^0.70.8",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.4.2"
+  },
+  "peerDependencies": {
+    "@animo-id/react-native-ble-didcomm": "^0.4.1",
+    "@aries-framework/core": "^0.3.2",
+    "react-native": "^0.70.6"
```

Comment:
The only requirement from react native is the event emitter, which was already in 0.60.0 so we could go even lower.

Reply:
I wsa think that we should keep the versions the same or close between the SDK and this transport, to prevent any issues and make any failures easier to debug

Reply:
@blu3beri @TimoGlastra

Reply:
The sdk can be lowered as well.

Reply:
`0.6.0`, then?